src/cookbook/design/orientation.md

```diff
@@ -135,7 +135,40 @@ class OrientationList extends StatelessWidget {
   <img src="/assets/images/docs/cookbook/orientation.gif" alt="Orientation Demo" class="site-mobile-screenshot" />
 </noscript>
 
+## Locking device orientation
```

Comment:
I didn't add a number in this case because it was after the "interactive example" as an extra, and not part of the step-by-step process described at the top.